util/src/main/java/io/kubernetes/client/util/WebSocketCall.java

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright (C) 2014 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+// This has been cloned from the okhttp-ws package so that I could remove
+// the requirement that websockets use the "GET" method
```

Comment:
Actually GET is a requirement. I suggest we remove this file and only support GET. from rfc6455:   

> 
> The method of the request MUST be GET, and the HTTP version MUST
>         be at least 1.1.
>